### PR TITLE
Added default testinfra tests

### PR DESCRIPTION
**Pull request by bdero**
Note: I'm not sure if it's possible to do conditional directories/files with cookiecutter. If it is, the tests file should be added when the author is MIT (or testinfra is otherwise enabled).

### CODE REVIEW

File: {{ cookiecutter.formula_name }}-formula/{{cookiecutter.formula_name}}/tests/test_install.sls [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{{ '{%' }} set package = salt.pillar.get('{{ cookiecutter.formula_name }}:install:pkgs') {{ '%}' }}
+
+test_installed_package:
+  testinfra.package:
+    - name: {{ '{{' }} package {{ '}}' }}
+    - is_installed: True
```
Comment (blarghmatey) [inline {{ cookiecutter.formula_name }}-formula/{{cookiecutter.formula_name}}/tests/test_install.sls:7]: I would change this to use the data from the map.jinja file. Something like:

``` salt
{{ '{%' }} from "{{cookiecutter.formula_name}}/map.jinja" import {{ cookiecutter.formula_name }} with context {{ '%}' }}

{{ '{%' }} for pkg in {{cookiecutter.formula_name}}.pkgs {{ '%}' }}
{% raw %}test_{{pkg}}_is_installed:
  testinfra.package:
    - name: {{ pkg }}
    - is_installed: True
{% endraw %}{% endfor %}
```

Comment (bdero) [inline {{ cookiecutter.formula_name }}-formula/{{cookiecutter.formula_name}}/tests/test_install.sls:7]: Thanks, I addressed this comment.

**Comment by bdero** [comment]
Let me know if you have any more thoughts @blarghmatey

**Comment by blarghmatey** [comment]
👍

**Pull request merged**